src/components/Header/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import { HeaderBar, HeaderDiv, BlankDiv, DMLogin } from './Header.styled';
+
+import { MenuBtn } from './Button/Menu/MenuBtn.jsx';

```

Comment:
You can change the export of your components to be something like: `export default MenuBtn` and then you can change this to: 

```suggestion
import MenuBtn from './Button/Menu/MenuBtn.jsx';
```

Reply:
We usually a have a default export for each component, and as you already create them, I think this is a good change to add